package.json

```diff
@@ -20,8 +20,10 @@
         "@types/node": "^20.6.0",
         "class-variance-authority": "^0.7.0",
         "firebase": "^10.5.2",
+        "hamburger-react": "^2.5.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
+        "react-icons": "^5.0.1",
```

Comment:
Was there an icon that couldn't be found in `@heroicons/react`? We already have an icon package, just don't want to have duplicates

Reply:
The icons from heroicons were either missing or way too different compared to the Figma file. I moved to using react-icons for that reason, but I could extract the icon SVGs from the Figma file and use those if need be!

Reply:
@ABCodez Its good then, just wanted to know why we needed another icons package. We can leave it as is.